src/Annotation/ApiSubresource.php

```diff
@@ -36,6 +39,8 @@ public function __construct($maxDepth = null)
     {
         if (!\is_array($maxDepth)) { // @phpstan-ignore-line
```

Comment:
Shouldn't we modify the PHPDoc?

Reply:
Not really as we want to keep the auto documentation as we did in https://github.com/api-platform/core/blob/main/src/Annotation/ApiResource.php